### PR TITLE
Polls for November 8, 2021

### PR DESCRIPTION
**Pull request by mkrorbrkr**
@LongForWisdom @prose11 

Polls for D3M parameter changes, WBTC-B Onboarding, RWF-001 budget and IS-001 onboarding

### CODE REVIEW

File: governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP39 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+With [MIP40c3-SP12](https://forum.makerdao.com/t/mip40c3-sp12-modify-core-unit-budget-rwf-001/7985) the Real-World Finance team was able to expand the team, sustain legal costs related to RWA, onboard more RWA and develop FP&A. This new budget extends the previous one in 2022.
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP40/MIP40c3-Subproposals/MIP40c3-SP39.md)
```
Comment (Hipogeo) [inline governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md:33]: Link should probably be: https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP40/MIP40c3-Subproposals/MIP40c3-SP39.md

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+The overall goal of the Immunefi Security CU is to secure the Maker ecosystem through multiple angles by leveraging its existing and growing security community and expertise. This involves dedicated services to the Maker ecosystem, effectively establishing a security partnership with the DAO. The CU will also aim to complement and collaborate with existing and future CUs instead of competing with them, leveraging existing processes, infrastructure, and information, among others, in order to deliver greater value to the Maker ecosystem.
+
+As a pioneer in the DeFi space with billions in funds in the protocol, the Maker ecosystem is an attractive target for blackhat hackers who look to steal funds or disrupt the protocol. With millions lost regularly in the cryptocurrency space, it is likely that the Maker protocol, its key people, and its key infrastructure, are targeted on a regular basis.
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP39/MIP39c2-Subproposals/MIP39c2-SP24.md)
```
Comment (Hipogeo) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:37]: Link should probably be:
https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP39/MIP39c2-Subproposals/MIP39c2-SP24.md

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+MIP40c3-SP42 adds the budget for Core Unit IS-001: Immunefi Security. It contains:
+
+* Total Budget Cap: The hard limit voted on by Governance
+* First-month forecast: The actual first month estimated expenses
+* Regular monthly forecast
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP40/MIP40c3-Subproposals/MIP40c3-SP42.md)
```
Comment (Hipogeo) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:52]: Link should probably be:
https://github.com/makerdao/mips/blob/9fd627c3fdfe4f2b8276a2efaefab59bb25239b5/MIP40/MIP40c3-Subproposals/MIP40c3-SP42.md

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+* Escrow Wallet mechanism
+
+The IS Core Unit follows the Sustainable Ecosystem Scaling Core Unit (SES-001) proposal for MKR Vesting.
+
+MKR incentives have been determined based on the Program discussed here. This is a 3-year vesting plan with 1-year cliff vest.
+
+The IS core unit has been working under the incubation program of the SES since October 2021. The members of this core unit will take that as the start date for the vesting schedule.
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP40/MIP40c3-Subproposals/MIP40c3-SP41.md)
```
Comment (Hipogeo) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:73]: Link should probably be:
https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP40/MIP40c3-Subproposals/MIP40c3-SP41.md

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp41-immunefi-security-core-unit-mkr-budget-is-001/10814)
+
+### MIP41c4-SP25
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+MIP41c4-SP25 adds @travinimmunefi as the Facilitator for Core Unit IS-001: Immunefi Security.
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP41/MIP41c4-Subproposals/MIP41c4-SP25.md)
```
Comment (Hipogeo) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:84]: Link should probably be:
https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP41/MIP41c4-Subproposals/MIP41c4-SP25.md

**Review by Hipogeo — APPROVED** [review]
Looks all good to me

File: governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md (hunk 1 above)
Comment (mkrorbrkr) [reply governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md:33]: I'm not sure what the difference is here. Other than linking to a branch that's not the master branch. Am I missing something?

**Review by Hipogeo — DISMISSED** [review]
Seems all good to go!

File: governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md (hunk 1 above)
Comment (blimpa) [reply governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md:33]: Hey, @mkrorbrkr :) The Cannonical Proposal Version is the latest commit, which can be found here:

image*

image*

\* For illustration purposes only. Disregard those hashes

File: governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP39 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+With [MIP40c3-SP12](https://forum.makerdao.com/t/mip40c3-sp12-modify-core-unit-budget-rwf-001/7985) the Real-World Finance team was able to expand the team, sustain legal costs related to RWA, onboard more RWA and develop FP&A. This new budget extends the previous one in 2022.
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md:30]: I'd probably link to [the MIPs Portal](https://mips.makerdao.com/mips/details/MIP40c3SP12) here. Also, formatting: This should be cursive

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
+---
+title: Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021
+summary: MIP39c2-SP24 adds Core Unit IS-001 - Immunefi Security
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:3]: "This MIP Set adds the Immunefi Security Core Unit (IS-001)"

Not really happy with that though, and we haven't been super consistent with these. Maybe "onboards" is better?

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+## Review
+
+The community may vote in this poll to express support or opposition to Adding Immunefi Security Core Unit - IS-001 being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP24
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+The overall goal of the Immunefi Security CU is to secure the Maker ecosystem through multiple angles by leveraging its existing and growing security community and expertise. This involves dedicated services to the Maker ecosystem, effectively establishing a security partnership with the DAO. The CU will also aim to complement and collaborate with existing and future CUs instead of competing with them, leveraging existing processes, infrastructure, and information, among others, in order to deliver greater value to the Maker ecosystem.
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:32]: Formatting: use _cursive_ for the summary

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP39/MIP39c2-Subproposals/MIP39c2-SP24.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP24)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp24-adding-immunefi-security-core-unit-is-001/10810)
+
+### MIP40c3-SP42
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+MIP40c3-SP42 adds the budget for Core Unit IS-001: Immunefi Security. It contains:
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:45]: Formatting: use _cursive_ for the summary

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP40/MIP40c3-Subproposals/MIP40c3-SP42.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP42)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp42-adding-immunefi-security-core-unit-budget-is-001/10813)
+
+### MIP40c3-SP41
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+MIP40c3-SP41 adds the MKR Incentive Plan budget for Core Unit IS-001: Immunefi Security. It contains:
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:60]: Formatting: use _cursive_ for the summary

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP40/MIP40c3-Subproposals/MIP40c3-SP41.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP41)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp41-immunefi-security-core-unit-mkr-budget-is-001/10814)
+
+### MIP41c4-SP25
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+MIP41c4-SP25 adds @travinimmunefi as the Facilitator for Core Unit IS-001: Immunefi Security.
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:81]: Formatting: use _cursive_ for the summary

File: governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md (hunk 6 above)
Comment (mkrorbrkr) [reply governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md:30]: Fixed. I just copied over the link that was used in the MIPs Portal doc but agree this is better

File: governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md (hunk 1 above)
Comment (mkrorbrkr) [reply governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md:33]: Ah thanks Pablo, that makes sense now :)

File: governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md (hunk 7 above)
Comment (mkrorbrkr) [reply governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md:3]: Yeah I agree onboarding is better - I copied the wording from the DUX poll fwiw https://github.com/makerdao/community/blob/master/governance/polls/Ratification%20Poll%20for%20Development%20%26%20UX%20Core%20Unit%2C%20DUX-001%20-%20September%2013%2C%202021.md

**Comment by mkrorbrkr** [comment]
@blimpa @Hipogeo thanks for the tips. Should all be correct now

File: governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md (hunk 6 above)
Comment (blimpa) [reply governance/polls/Ratification Poll for Modify Core Unit Budget, RWF-001 (MIP40c3-SP39) - November 8, 2021.md]: Yes. Looks like I missed that one on the proposal :[

File: governance/polls/Ratification Poll for Immunefi Security Core Unit (IS-001) - November 8, 2021.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021
-summary: MIP39c2-SP24 adds Core Unit IS-001 - Immunefi Security
+title: Ratification Poll for Immunefi Security Core Unit (IS-001) - November 8, 2021
+summary: MIP39c2-SP24 onboards Core Unit IS-001 - Immunefi Security
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Immunefi Security Core Unit (IS-001) - November 8, 2021.md:3]: "This MIP Set onboards the Immunefi Security Core Unit (IS-001)"

Comment (blimpa) [reply governance/polls/Ratification Poll for Immunefi Security Core Unit (IS-001) - November 8, 2021.md:3]: Otherwise, I think it looks good :)

**Pull request merged**